modules/api/src/main/java/org/eclipse/xpanse/api/config/CaffeineCacheConfig.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.api.config;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import java.util.concurrent.TimeUnit;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * The configuration class of Caffeine Cache.
+ */
+@Configuration
+public class CaffeineCacheConfig {
+
+    @Value("${caffeine.cache.expire.time:60}")
```

Comment:
It is better to add time uint in the configuration item name, like 'caffeine.cache.expire.minutes'.